request_a_govuk_domain/request/templates/simple_history/_object_history_list.html

```diff
@@ -0,0 +1,91 @@
+    }
+
+    h2 {
+        margin-top: 2%;
+    }
+</style>
+<table id="change-history" class="table table-bordered table-striped full_width">
+    <thead>
+    <tr>
+        <th scope="col">{% trans 'Date/time' %}</th>
+        <th scope="col">{% trans 'User' %}</th>
+        <th scope="col">{% trans 'Status' %}</th>
+
+    </tr>
+    </thead>
+    <tbody>
+    {% with filtered_actions=action_list %}
+        {% for action in filtered_actions %}
+            {% should_display action as should_display %}
+            {% if should_display %}
+                <tr>
+                    <td>
+                        <a href="{% url opts|admin_urlname:'simple_history' object.pk action.pk %}">{{ action.history_date }}</a>
+                    </td>
+                    <td>
+                        {% if action.history_object.last_updated_by %}
+                            {% url admin_user_view action.history_object.last_updated_by.id as admin_user_url %}
+                            {% if admin_user_url %}
+                                <a href="{{ admin_user_url }}">{{ action.history_object | owner_filter }}</a>
+                            {% else %}
+                                {{ action.history_object.last_updated_by }}
+                            {% endif %}
+                        {% else %}
+                            {% trans "-" %}
+                        {% endif %}
+                    </td>
+                    <td>
+                        {{ action.history_object.status | capfirst }}
+                    </td>
+                </tr>
+            {% endif %}
+        {% endfor %}
+    {% endwith %}
+    </tbody>
+</table>
+
+<h2>Full event history</h2>
+<table id="change-history" class="table table-bordered table-striped full_width">
+    <thead>
+    <tr>
+        <th scope="col">{% trans 'Date/time' %}</th>
+        <th scope="col">{% trans 'Event' %}</th>
+        <th scope="col">{% trans 'Changed by' %}</th>
+        <th scope="col">{% trans 'Change fields' %}</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for action in action_list %}
+        <tr>
+            <td>
+                <a href="{% url opts|admin_urlname:'simple_history' object.pk action.pk %}">{{ action.history_date }}</a>
+            </td>
+            <td>{{ action.get_history_type_display }}</td>
+            <td>
+                {% if action.history_object.last_updated_by %}
+                    {% url admin_user_view action.history_object.last_updated_by.id as admin_user_url %}
+                    {% if admin_user_url %}
+                        <a href="{{ admin_user_url }}">{{ action.history_object.last_updated_by }}</a>
+                    {% else %}
+                        {{ action.history_object.last_updated_by }}
+                    {% endif %}
+                {% else %}
+                    {% trans "-" %}
+                {% endif %}
+            </td>
+            <td>
+                <i>{{ action | changed_fields }}</i>
```

Comment:
Why `<i>`? I don't think it's needed. It actually makes the text harder to read. Maybe you could also use something to join using commas as opposed to displaying the raw string. Better `abc, def` than `['abc', 'def']`